### DISCUSSION TITLE
How to deploy crd for "monitoring.coreos.com/v1"?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12053] [Q&A] [answered]
error: unable to recognize "deploy/examples/monitoring/service-monitor.yaml": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"

**Comment by galexrt** [comment, marked as answer]
You need to have the Prometheus Operator deployed in your cluster. This doc page about Prometheus monitoring should get you into the right direction: https://rook.io/docs/rook/v1.10/Storage-Configuration/Monitoring/ceph-monitoring/?h=monitoring
